src/SonarQube/SolutionBinding.sqconfig

```diff
@@ -1,10 +1,4 @@
 {
   "ServerUri": "https://dory.sonarsource.com/sonarqube/",
-  "ProjectKey": "sonarlint-visualstudio",
-  "Profiles": {
-    "CSharp": {
-      "ProfileKey": "cs-c-qp-for-sonarsource-73109",
-      "ProfileTimestamp": "2016-04-08T10:15:44+01:00"
-    }
-  }
+  "ProjectKey": "sonarlint-visualstudio"
```

Comment:
This might be a mistake on my side. I did the resync with version 2.1, and not the snapshot version.